terrautils/sensors.py

```diff
@@ -118,78 +116,92 @@
               "fixed_metadata_datasetid": "TBD"
         },
 
-        'rgb_fullfield': {
+        'fullfield': {
+            'display': 'Full Field Stitched Mosaics',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{filename}',
             'pattern': '{sensor}_lv1_{station}_{date}{opts}.tif',
         },
 
-        'flirIrCamera': {
+        'vnir_netcdf': {
+            'display': 'VNIR Hyperspectral NetCDFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
-            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
+            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.nc',
         },
 
-        'stereoTop_geotiff': {
+        'swir_netcdf': {
+            'display': 'SWIR Hyperspectral NetCDFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
-            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
+            'pattern': '{sensor}_L1_{station}_{timestamp}{opts}.nc',
         },
 
-        'stereoTop_canopyCover': {
+        'stereoTopRGB_geotiff': {
+            'display': 'stereoTop RGB GeoTIFFs',
             'template': '{base}/{station}/Level_1/'
                         '{sensor}/{date}/{time}/{filename}',
-            'pattern': '{sensor}_lv1_{station}_{timestamp}{opts}.tif',
+            'pattern': '{sensor}_L1_{station}_{timestamp}{opts}.tif',
         },
 
+        'stereoTopRGB_canopyCover': {},
+
         'texture_analysis': {
```

Comment:
Just to be clear I think that these csv files can also be excluded from the filesystem if they are added to BETYdb.